presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -49,6 +49,50 @@ bool SystemConfig::httpServerReusePort() const {
   return opt.value_or(kHttpServerReusePortDefault);
 }
 
+int SystemConfig::httpServerHttpsPort() const {
+  return requiredProperty<int>(std::string(kHttpServerHttpsPort));
+}
+
+bool SystemConfig::enableHttps() const {
+  auto opt = optionalProperty<bool>(std::string(kHttpServerHttpsEnabled));
+  return opt.value_or(kHttpServerHttpsEnabledDefault);
+}
+
+// This config control what cipher suites are supported by Native workers for
```

Comment:
Nit: This config controls

Reply:
Thanks. Let me submit another PR to fix.